frontend/src/scenes/organization/Settings/VerifiedDomains/AddDomainModal.tsx

```diff
@@ -33,32 +33,32 @@ export function AddDomainModal(): JSX.Element {
     }
 
     return (
-        <LemonModal onCancel={handleClose} visible={addModalShown} destroyOnClose>
-            <section>
-                <h5>Add authentication domain</h5>
-
-                <Input
-                    placeholder="posthog.com"
-                    autoFocus
-                    value={newDomain}
-                    onChange={(e) => setNewDomain(e.target.value)}
-                    onPressEnter={handleSubmit}
-                />
-                {submitted && errored && (
-                    <span className="text-danger text-xs">
-                        Please enter a valid domain or subdomain name (e.g. my.posthog.com)
-                    </span>
-                )}
-                <div style={{ display: 'flex', justifyContent: 'flex-end' }}>
-                    <LemonButton
-                        type="primary"
-                        disabled={newDomain === '' || (submitted && errored) || verifiedDomainsLoading}
-                        onClick={handleSubmit}
-                    >
-                        Add domain
-                    </LemonButton>
-                </div>
-            </section>
+        <LemonModal
+            onClose={handleClose}
+            isOpen={addModalShown}
+            title="Add authentication domain"
+            footer={
```

Comment:
We have some that take `footer` as a prop and some that have `Lemon.Footer` as a child...

Do we need both approaches? Does storybook teach us how to choose?

Reply:
What happens if I add both :)

Reply:
They are sadly necessary due to forms but for sure should be better documented. I'll update the storybook to be clear about it